Week3/homework/step2-3.js

```diff
@@ -4,10 +4,9 @@
 function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line prefer-const
   let result = '';
-
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
+  for (let i = 0; i < num; i++) {
```

Comment:
Correct. However, it would have been nice if you had an `if` condition to check for values of `num` less than or equal to 0. 🙂